config/config_env.go

```diff
@@ -157,11 +161,17 @@ func reporterConfigFromEnv() (*ReporterConfig, error) {
 
 	host := jaeger.DefaultUDPSpanServerHost
 	if e := os.Getenv(envAgentHost); e != "" {
+		if ep := os.Getenv(envEndpoint); ep != "" {
```

Comment:
how about we read the envEndpoint once and use `ep` in all the checks?

Reply:
ack